### PR TITLE
chore(leviton): add debugging to acquisuite driver

### PR DESCRIPTION
**Pull request by camreeves**
Super simple. Can someone approve asap, thanks.

### CODE REVIEW

**Review by w-le — DISMISSED** [review]
lgtm

File: drivers/leviton/acquisuite.cr [hunk 1]
```diff
@@ -31,6 +31,15 @@ class Leviton::Acquisuite < PlaceOS::Driver
...
 
   def receive_webhook(method : String, headers : Hash(String, Array(String)), body : String)
+    if @debug_webhook
+      logger.debug { "Received Webhook"}
+      logger.debug { "Method:" }
+      logger.debug { method.inspect }
+      logger.debug { "Headers:" }
+      logger.debug { headers.inspect }
+      logger.debug { "Body:" }
+      logger.debug { body.inspect }
```
Comment (stakach) [inline drivers/leviton/acquisuite.cr:41]:
```suggestion
      logger.debug do
        "Received Webhook\n" +
        "Method: #{method.inspect}\n" +
        "Headers:\n#{headers.inspect}\n" +
        "Body:\n#{body.inspect}"
      end
```

**Review by stakach — APPROVED** [review]
multiple debug lines aren't very performant

File: drivers/leviton/acquisuite.cr (hunk 1 above)
Comment (camreeves) [reply drivers/leviton/acquisuite.cr]: Great, thanks!

**Review by stakach — APPROVED** [review]
LGTM

**Comment by camreeves** [comment]
Sorry @stakach one more little change

**Review by stakach — APPROVED** [review]

**Pull request merged**